docs/StardustDocs/topics/schemasImportSqlGradle.md

```diff
@@ -0,0 +1,126 @@
+[//]: # (title: Import SQL metadata as a schema in Gradle project)
+
+<!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Schemas-->
+
+Each SQL database keeps the metadata for all the tables, and 
+this metadata could be used for the schema generation.
+
+**NOTE:** Visit the [page](readSqlDatabases.md) to see how to set up all Gradle dependencies for your project.
+
+### With `@file:ImportDataSchema`
+
+To generate schema for existing SQL table,
+you need to define a few parameters to establish JDBC connection:
+URL, username, and password.
+
+Also, the `tableName` parameter could be specified.
+
+You should also specify the name of the generated Kotlin class 
+as a first parameter of annotation `@file:ImportDataSchema`.
+
+```kotlin
+@file:ImportDataSchema(
+    "ActorSchema",
```

Comment:
might be clearer to add names to all of the arguments here if we ever decide to change the order